client/src/assets/icons/fail.tsx

```diff
@@ -20,16 +20,12 @@ function RedFail() {
           r='95'
           stroke='(var--primary-color)'
           strokeDasharray='null'
-          strokeLinecap='null'
-          strokeLinejoin='null'
```

Comment:
Should these have been removed?

Reply:
![image](https://user-images.githubusercontent.com/34807532/122665990-57b3d600-d1c8-11eb-8827-3fd844b596e0.png)

I had it removed because `null` isn't a valid attribute for these properties. I assumed that it is being either set to default by browsers or somewhere else. That said, I'm not sure if these should have been removed. I did it because I did not know what else to do.